omod/src/main/webapp/resources/scripts/conditionlist/controllers/manageconditions.controller.js

```diff
@@ -38,6 +38,10 @@ function ManageConditionsController($scope, RestfulService, CommonFunctions) {
             self.saveCondition(condition);
         }
 
+    self.redirectToEditCondition = self.redirectToEditCondition || function(baselink,condition) {
+        window.location= baselink+'condition='+JSON.stringify(condition)+'&';
```

Comment:
How would a sample value for window.location look like?

Reply:
https://appendto.com/2016/04/javascript-redirect-how-to-redirect-a-web-page-with-javascript/